apps/src/templates/PauseButton.jsx

```diff
@@ -60,27 +68,27 @@ class PauseButton extends React.Component {
 const styles = {
   icon: {
     lineHeight: 'inherit',
-    color: color.white
+    fontSize: '2.8em'
   },
   container: {
     width: 40,
     height: 40,
     lineHeight: '40px',
     textAlign: 'center',
     verticalAlign: 'middle',
-    display: 'inline-block'
+    display: 'flex',
+    justifyContent: 'center'
   },
   button: {
     minWidth: 0,
     padding: 0,
-    borderRadius: '100%'
+    borderRadius: '100%',
+    backgroundColor: color.white
```

Comment:
Why did `backgroundColor` need to change here?

Reply:
Previously we had white icons over orange and cyan buttons. Now we'd have orange and cyan icons over white buttons. This is due to the black and white space of the icons being inverted.